### ISSUE TITLE
Add imports don't show up on first quick fix

### ISSUE THREAD

**Issue by rchiodo** [microsoft/pylance-release#5805]
![image](https://github.com/microsoft/pylance-release/assets/19672699/339abcc0-8c58-473b-9169-f270454e68cf)

This happens on WSL when I open the pylance-release/testing/single/codeactions.py. If I bring up code actions for the second error it works, and then if I go back to the `os` error, it  now shows this:

![image](https://github.com/microsoft/pylance-release/assets/19672699/1ea0b2a1-3580-49c9-a857-939bd9e93d54)

**Comment by heejaechang** [comment]
this happens regardless how long you waited before doing the first code action?

**Comment by rchiodo** [comment]
Let me try again to make sure, but I believe so.

**Comment by rchiodo** [comment]
Yes it does. I have to do this first though:

1. Open VS code
2. Open the codeActions.py file
3. Put the cursor in the `os` line
4. Close codeActions.py file
5. Reload VS code
6. Open codeActions.py file
7. Try quick fix

No matter how long I wait, it shows this:
![image](https://github.com/microsoft/pylance-release/assets/19672699/db27b3ec-7f91-4b28-aab4-38357fb6542a)

If I move the cursor it works again.

My guess is that VS code isn't reasking. It must be asking as we're still loading the index information.

**Comment by heejaechang** [comment]
my guess is the first code action request came before indexing is done, so we returned code action response without `import os` and that response is cached until new trigger such as caret move or text change or etc. so, control+. won't show `import os` until that happen.

I guess we could trigger whatever that would trigger new code action request when indexing is done (if such thing exists like semantic refresh or etc)

**Comment by heejaechang** [comment]
hmmm.. it doesn't repro for me somehow in WSL. if you enable `logLeve: Trace` and do ctrl+. after `index is done` log, is it still happening? it might not happening to me since I don't have much packages installed so `indexing` is relatively cheap.

**Comment by rchiodo** [comment]
CTRL+. seems to work. It's just the quick action lightbulb that has the wrong data. And if I do CTRL+. and then click on the lightbulb, the lightbulb then has the correct data.

**Comment by heejaechang** [comment]
I see. then `ctrl+.` must doing new `code action request` but existing `lightbulb` is using cached info.

just looked through LSP spec on code action -https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_codeAction - but couldn't find any `refresh` or similar request. looks like when to refresh code action info seems baked in the code itself.

**Comment by rchiodo** [comment]
No other complaints so just closing.
